### ISSUE TITLE
`valid-describe` false positive with `.each`

### ISSUE THREAD

**Issue by SimenB** [jest-community/eslint-plugin-jest#468]
Using `23.0.2`.

```ts
describe.each([])('%s', s => {});
```

**Comment by G-Rath** [comment]
@SimenB Cheers - I've confirmed this is fixed as I expected.

I guess the rebasing of the commit on master didn't trigger a release?

It should be released when #469 is merged

**Comment by SimenB** [comment]
Can confirm it's fixed in 23.0.3 👍
